### PR TITLE
systemd: refactor property parsers for int values in CurrentTasksCount, etc.

### PR DESCRIPTION
**Pull request by anonymouse64**
As suggested by Maciej, Michael and Samuele in snapcore/snapd#10354.

### CODE REVIEW

File: systemd/systemd.go [hunk 1]
```diff
@@ -615,44 +615,50 @@ func (s *systemd) getPropertyStringValue(unit, key string) (string, error) {
 	return strings.TrimSpace(splitVal[1]), nil
 }
 
-func (s *systemd) CurrentTasksCount(unit string) (int, error) {
-	tasksStr, err := s.getPropertyStringValue(unit, "TasksCurrent")
+func (s *systemd) getPropertyUintValue(unit, key string) (uint64, bool, error) {
```
Comment (mvo5) [inline systemd/systemd.go:618]: (bikeshed) Maybe just return `(uint64, err)` here and have something like PropertyUnavailableError or just ErrNotExists or something?

**Review by mvo5 — COMMENTED** [review]
Thanks for this refactor. One tiny suggestion about the api inline

**Review by stolowski — APPROVED** [review]
LGTM

**Review by mvo5 — APPROVED** [review]
Nice, thank you!

**Pull request merged**